main.js

```diff
@@ -10,4 +11,288 @@ function closeMenu() {
+firstProjectBtn.innerHTML = 'See Project';
+firstProjectBtn.classList.add('project1');
+firstProjectBtn.href = '#';
+const multistoriesContainer = document.createElement('div');
+multistoriesContainer.classList.add('multistories');
+
+function createFirstProject() {
+  firstProjects.forEach((firstProject) => {
+    const recentWork = document.createElement('div');
+    recentWork.classList.add('recent-work');
+    multistoriesContainer.innerHTML = `<h2>${firstProject.name}</h2>
+          <p>
+            ${firstProject.description}
+          </p>
+          <ul class="languages-1">
+            <li class="html">Html</li>
+            <li class="css">Css</li>
+            <li class="javascript">Javascript</li>
+          </ul>`;
+    recentWork.innerHTML = `<div class="placeholder-multistories">
+          <img src="./img/placeholder.png" class="placeholder" alt="placeholder-1" />
+        </div>`;
+    multistoriesContainer.appendChild(firstProjectBtn);
+    recentWork.appendChild(multistoriesContainer);
+    mainTagContainer.appendChild(recentWork);
+  });
+}
+
+const firstProjectContainer = document.createElement('div');
+firstProjectBtn.addEventListener('click', () => {
+  firstProjectContainer.style.display = 'block';
+  firstProjectContainer.innerHTML = `<div class="main_popup_container" id="modal-3">
+    <div class="popup_heading_container">
+
+    <h2 class="main_popup_heading">Multi Post Stories</h2>
+
+    <div>
+    
+    <img class="close_first" src="./img/cross.png" onclick = "closePopupWindow()">
+    
+    </div>
+
+    </div>
+
+    <ul class="popup_language_box">
+    
+    <li class="language_box_item">HTML</li><li class="language_box_item">CSS</li><li class="language_box_item">Ruby On Rails</li>
+
+    </ul>
+    <div class="popup_grid_container">
+    
+    <img src="./img/popup-img-desktop.png" class="popup_img">
+
+
+      <p class="popup_para">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec auctor, nisl eget ultricies lacinia, nunc nisl aliquam nisl, eget aliquam nunc nisl eget nunc. Donec auctor, nisl eget ultricies lacinia, nunc nisl aliquam nisl, eget aliquam nunc nisl eget nunc.</p>
+
+
+      <div class="popup_button_main_container">
+
+
+        <div class="popup_buttons_container">
+        
+        <a class="popup_buttons" href="https://bilalrajput09.github.io/My-Portfolio/">See Live</a>
+        
+        <span class="popup_icons">
+
+            <img src="./img/see-live.png">
+            
+        </span>
+
+        </div>
+        <div class="popup_buttons_container">
+        
+        <a class="popup_buttons" href="https://github.com/bilalrajput09">See Source</a>
+
+        <span class="popup_icons">
+
+        <img src="./img/github-white-popup.png">
+
+        </span>
+        </div>
+      </div>
+    </div>
+    </div>`;
+  document.body.appendChild(firstProjectContainer);
+  backdrop.classList.remove('backdrop-hidden');
+});
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] As mentioned by the previous reviewer, please do not disable any linter. Fix what ever issues it is.